app/models/patron.rb

```diff
@@ -12,6 +20,27 @@ def key
     record['key']
   end
 
+  def first_name
+    fields['firstName']
+  end
+
+  def last_name
+    fields['lastName']
+  end
+
+  def status
+    if expired?
+      'Expired'
+    else
+      PATRON_STANDING.fetch(fields['standing']['key'], '')
+    end
+  end
+
+  def expired?
+    expired_date = fields['privilegeExpiresDate']
```

Comment:
Maybe we could extract `expired_date` to its own method, and have it do the same sort of date parsing we do in e.g. `Checkout#recalled_date`?